geomesa-tools/src/test/scala/org/locationtech/geomesa/tools/ConvertCommandTest.scala

```diff
@@ -8,267 +8,92 @@
-      |      ]
-      |    }
-      |  }
-      |}
-    """.stripMargin
+  val jsonInput = getClass.getResource("/convert/json-data.json").getFile
+  val jsonConf  = FileUtils.readFileToString(new File(getClass.getResource("/convert/json-convert.conf").getFile))
 
-  val testJSON =
-    """
-      |{
-      |	DataSource: { name: "example-json" },
-      |	Features: [
-      |		{
-      |			fid: 23623,
-      |			name: "Harry",
-      |			age: 20,
-      |			lastseen: "2015-05-06",
-      |			friends: "Will, Mark, Suzan",
-      |			talents: "patronus->10,expelliarmus->9",
-      |			lat: -100.236523,
-      |			lon: 23,
-      |			vis: "user"
-      |		},
-      |		{
-      |			fid: 26236,
-      |			name: "Hermione",
-      |			age: 25,
-      |			lastseen: "2015-06-07",
-      |			friends: "Edward, Bill, Harry",
-      |			talents: "accio->10",
-      |			lat: 40.232,
-      |			lon: -53.2356,
-      |			vis: "user"
-      |		},
-      |		{
-      |			fid: 3233,
-      |			name: "Severus",
-      |			age: 30,
-      |			lastseen: "2015-10-23",
-      |			friends: "Tom, Riddle, Voldemort",
-      |			talents: "potions->10",
-      |			lat: 3,
-      |			lon: -62.23,
-      |			vis: "user&admin"
-      |		}
-      |	]
-      |}
-    """.stripMargin
-  val testJSONFile = File.createTempFile("ConvertCommandTestData", ".json")
-  new PrintWriter(testJSONFile) { write(testJSON); close() }
-  val jsonConf =
-    """
-      |geomesa {
-      |  sfts {
-      |    "example-json" = {
-      |      attributes = [
-      |        { name = "id",   type = "Integer" }
-      |        { name = "name", type = "String"  }
-      |        { name = "age",  type = "Integer" }
-      |        { name = "geom", type = "Point"   }
-      |      ]
-      |    }
-      |  }
-      |  converters {
-      |    "example-json" = {
-      |      type         = "json"
-      |      id-field     = "$id"
-      |      feature-path = "$.Features[*]"
-      |      options {
-      |        line-mode = "multi"
-      |      }
-      |      fields = [
-      |        { name = "id",   json-type = "integer", path = "$.fid",  transform = "toString($0)"      }
-      |        { name = "name", json-type = "string",  path = "$.name",                                 }
-      |        { name = "age",  json-type = "integer", path = "$.age",                                  }
-      |        { name = "lat",  json-type = "double",  path = "$.lat",                                  }
-      |        { name = "lon",  json-type = "double",  path = "$.lon",                                  }
-      |        { name = "geom",                                         transform = "point($lon, $lat)" }
-      |      ]
-      |    }
-      |  }
-      |}
-    """.stripMargin
+  val inFormats = Seq(DataFormats.Csv, DataFormats.Tsv, DataFormats.Json)
+  val outFormats = DataFormats.values.filter(f => f != DataFormats.Null && f != DataFormats.Arrow).toSeq
```

Comment:
Does this do what we want?  

Would it be cleaner to call .toSet and then .- to remove things.  

Either that or I have it backwards.  It might be faster to say "val outFormats = Seq(DataFormats.Null, DataFormats.Arrow)